provision/acc_provision/templates/provision-config.yaml

```diff
@@ -2,21 +2,29 @@
 # Configuration for ACI Fabric
 #
 aci_config:
-  system_id: mykube             # Every opflex cluster must have a distict ID
+  system_id: mykube              # Every opflex cluster must have a distict ID
```

Comment:
there seem to quite a few unnecessary/unrelated changes in this (spacing, comments etc), can you remove them? This is a sample file, so for your testing please create a separate input config file instead of modifying this one.